include/zephyr/sys/spsc_pbuf.h

```diff
@@ -31,11 +31,19 @@ extern "C" {
 #define SPSC_PBUF_UTILIZATION_OFFSET 8
 
 /**@} */
+#ifdef CONFIG_DCACHE_LINE_SIZE
+#define Z_SPSC_PBUF_LOCAL_DCACHE_LINE CONFIG_DCACHE_LINE_SIZE
+#else
+#define Z_SPSC_PBUF_LOCAL_DCACHE_LINE 0
+#endif
```

Comment:
You probably want to check for `CONFIG_DCACHE_LINE_SIZE_DETECT` as well. See in include/zephyr/cache.h

Reply:
for runtime detection it is a bit tricky because i need static value since it is used for array size declaration.
What i did is to mention in spsc_pbuf cache line configuration that if runtime detection of data cache line size is used then this option must be set to max(local, remote).
I've also added support for getting line size from device tree.